test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/CommonTriageQuestionsControllerSpec.scala

```diff
@@ -154,36 +157,110 @@ class CommonTriageQuestionsControllerSpec
       draftReturn = draftReturn,
       subscribedDetails = sample[SubscribedDetails].copy(name = name),
       previousSentReturns = previousReturns,
-      amendReturnData = if (isAmend) Some(sample[AmendReturnData]) else None
+      amendReturnData =
+        if (isAmend)
+          Some(
+            sample[AmendReturnData].copy(
+              shouldDisplayGainOrLossAfterReliefs = true
```

Comment:
setting the flag to `true` here prevents the tests from testing the `false` to `true` behaviour when it is an amend. I'd leave this as it was and use the `withForceDisplayGainOrLossAfterReliefsForAmends` on `FillingOutReturn` where you need it in your tests